apps/book/management/commands/load_book_data.py

```diff
@@ -0,0 +1,40 @@
+from django.core.management.base import BaseCommand
+
+from apps.common.models import Province, District, Municipality
+from apps.book.factories import BookFactory, TagFactory, AuthorFactory, CategoryFactory
+from apps.publisher.factories import PublisherFactory
+
+
+class Command(BaseCommand):
+    help = 'Create some fake book data'
+
+    def handle(self, *args, **options):
+        # Create tags, authors can categories
+        tag1, tag2, tag3 = TagFactory.create_batch(3)
+        author1, author2, author3 = AuthorFactory.create_batch(3)
+        category1, category2, category3 = CategoryFactory.create_batch(3)
+
+        # Get province, district, municipality, make sure geo data is loaded
+        province = Province.objects.first()
+        district = District.objects.first()
+        municipality = Municipality.objects.first()
+        publisher1, publisher2, publisher3 = PublisherFactory.create_batch(
+            3, province=province, district=district, municipality=municipality
+        )
```

Comment:
Let's do something like this.
```suggestion
        municipality = Municipality.objects.first()
        if municipality is None:
            print('Make sure to load geo data first')
            return
            
        # Create tags, authors can categories
        tag1, tag2, tag3 = TagFactory.create_batch(3)
        author1, author2, author3 = AuthorFactory.create_batch(3)
        category1, category2, category3 = CategoryFactory.create_batch(3)

        # Get province, district, municipality, make sure geo data is loaded
        publisher1, publisher2, publisher3 = PublisherFactory.create_batch(
            3, province=municipality.district.province, district=municipality.district, municipality=municipality
        )
```